release/cloud_builder.sh

```diff
@@ -103,23 +103,8 @@ popd
 # doesn't help when the user doesn't have one.
 touch pilot/platform/kube/config
 
-# building //... results in dirtied files:
-# broker/pkg/model/config/mock_store.go
-# broker/pkg/platform/kube/crd/types.go
-# mixer/template/apikey/go_default_library_handler.gen.go
-# mixer/template/apikey/go_default_library_tmpl.pb.go
-# mixer/template/template.gen.go
-bazel build //pilot/...
-
-# bazel_to_go likes to run from dir with WORKSPACE file
-./bin/bazel_to_go.py
-# Remove doubly-vendorized k8s dependencies that confuse go
-rm -rf vendor/k8s.io/*/vendor
-
-# bazel_to_go.py dirties generated_files and lintconfig.json
-# it's easier to ask git to restore files than add
-# an option to bazel_to_go to not touch them
-git checkout generated_files
+# pull in outside dependencies
+make depend
```

Comment:
this can be removed if you like.  Now that the lock file is checked in, the standard vendor target ends up running depend.ensure.

Reply:
After rebasing I again tried removing "make depend".  Previously it got as far as mixer's push-docker (which calls mixer's Makefile [rather than the top-level Makefile] that doesn't try to deal with deps).  After the rebase it failed earlier while trying to run pilot's upload-istioctl.

Reply:
I see - seems like we still have some more work to do in the repo :)  My suggestion here can be reverted and we can resolve this problem as part of general cleanup.